audio_clip/model/esresnet/attention.py

```diff
@@ -0,0 +1,55 @@
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+
+from typing import Tuple
+
+
+class EvilBatchNorm2d(torch.nn.BatchNorm2d):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.running_var = None
+        self.running_mean = None
+
```

Comment:
Does this achieve the same as overriding `.eval()`?

Reply:
No the same things as in training